devtools/platform-descriptor-json/src/main/resources/templates/dockerfile-jvm.ftl

```diff
@@ -14,21 +14,30 @@
 # docker run -i --rm -p 8080:8080 quarkus/${project_artifactId}-jvm
 #
 ###
-FROM fabric8/java-alpine-openjdk8-jre:1.6.5
-ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
-ENV AB_ENABLED=jmx_exporter
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
```

Comment:
In https://access.redhat.com/containers/?tab=images#/registry.access.redhat.com/ubi8/ubi-minimal the recommended URL is `registry.redhat.io/ubi8/ubi-minimal` instead of `registry.access.redhat.com`

Reply:
See https://www.redhat.com/en/blog/transitioning-red-hat-container-registry

Reply:
ok, didn't know that. The native Dockerfile is also using `registry.access.redhat.com`

Reply:
Yeah, now it's the time to use the suggested one :wink:

Reply:
```
$docker pull registry.redhat.io/ubi8/ubi-minimal:8.1                                              
Error response from daemon: Get https://registry.redhat.io/v2/ubi8/ubi-minimal/manifests/8.1: unauthorized: Please login to the Red Hat Registry using your Customer Portal credentials. Further instructions can be found here: https://access.redhat.com/RegistryAuthentication
```

I know why it's recommended. But it's not user-friendly.